include/pcl_conversions/pcl_conversions.h

```diff
@@ -121,7 +118,9 @@ namespace pcl_conversions {
   {
     toPCL(header.stamp, pcl_header.stamp);
     // TODO(clalancette): Seq doesn't exist in the ROS2 header
-    // anymore.  Do we need to do anything here?
+    // anymore.  wjwwood suggests that we might be able to get this
+    // information from the middleware in the future, but for now we
+    // just set it to 0.
     //pcl_header.seq = header.seq;
```

Comment:
:+1: for the new comment. This line can be removed though given that header will not have a seq field in the future